programs/voter-stake-registry/src/instructions/log_voter_info.rs

```diff
@@ -50,7 +50,7 @@ pub fn log_voter_info(
         let voting_mint_config = &registrar.voting_mints[deposit.voting_mint_config_idx as usize];
         let locking_info = (seconds_left > 0).then(|| LockingInfo {
             amount: deposit.amount_locked(curr_ts),
-            end_timestamp: (lockup.kind != LockupKind::Constant).then_some(end_ts),
```

Comment:
This caused compiler errors for rustc `1.60.0` and `nightly-2022-04-01`

Reply:
Yeah, bool::then_some() was added in 1.62.